pkg/controller/appsync/appsync.go

```diff
@@ -44,8 +49,98 @@ func New(config *config.AppSyncConfig, db *database.Database, h *render.Renderer
 
 // HandleSync performs the logic to sync mobile apps.
 func (c *Controller) HandleSync(ctx context.Context) http.Handler {
+	type AppSyncResult struct {
+		OK     bool    `json:"ok"`
+		Errors []error `json:"errors,omitempty"`
+	}
+
 	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
-		// TODO(whaught): implement this
-		controller.InternalError(w, r, c.h, errors.New("not implemented"))
+		logger := logging.FromContext(ctx).Named("appsync.HandleSync")
+
+		client := http.Client{Timeout: c.config.Timeout}
+		resp, err := client.Get(c.config.AppSyncURL)
+		if err != nil {
+			controller.InternalError(w, r, c.h, err)
+			return
+		}
+
+		defer resp.Body.Close()
+		body, err := ioutil.ReadAll(io.LimitReader(resp.Body, c.config.FileSizeLimit))
```

Comment:
I think it'd be better if we didn't have this intermediate []byte data structure:

```go
var apps AppsResponse
if err := json.NewDecoder(io.LimitReader(resp.Body, c.config.FileSizeLimit).Decode(&apps); err != nil {
  // ...
}
```